.c8rc

```diff
@@ -1,6 +1,17 @@
 {
   "all": true,
   "include": ["out/**"],
-  "exclude": ["**/node_modules/**", "out/test/"],
+  "exclude": [
+    "**/node_modules/**",
+    "out/test/",
+    "out/api-client/",
+    "out/utils/env.js",
+    "out/utils/telemetry.js",
+    "out/utils/types.js",
+    "out/tree/",
+    "out/services/analyticsStreamApiService.js",
+    "out/features/issuance/json-schema",
+    "out/config/"
```

Comment:
IMO if they're not covered – it's fine, I'd say it's better to have lower coverage than to maintain this list.

Plus most of these things _can_ be covered, we're just lazy right now

Reply:
some of them r redundant because there is nothing to cover 
```js
   "out/utils/env.js",
    "out/utils/telemetry.js",
    "out/utils/types.js",
```

Reply:
Yeah you can keep those here, but, personally, since we're not using "coverage statistics" anywhere – we probably should just not care about this config.

Reply:
reverted config